manifests/volume.pp

```diff
@@ -102,7 +102,7 @@
   if $binary{
     # we need the Gluster binary to do anything!
 
-    if ! member( split( $::gluster_volume_list, ',' ), $title ) {
+    if ! member( split( "$::gluster_volume_list", ',' ), $title ) {
```

Comment:
To comply with Puppet link checks, this should be `"${::gluster_volume_list}"`.  Variables must use brace notation inside of quoted strings.